### DISCUSSION TITLE
clip intensity in transform

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#4228] [Q&A] [answered]
Thanks for the great job.
I would like to achieve 'clip(min, max)' operation in transform, which means input image with intensity below min should be set to min and above max  should be max.
I dont find relevent transforms in MONAI, If I miss something?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @YingdiZhang ,

I think you can use `ThresholdIntensity` to solve this problem:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/intensity/array.py#L701

Thanks.
